docs/flexbox.md

```diff
@@ -9,10 +9,37 @@ You will normally use a combination of `flexDirection`, `alignItems`, and `justi
 
 > Flexbox works the same way in React Native as it does in CSS on the web, with a few exceptions. The defaults are different, with `flexDirection` defaulting to `column` instead of `row`, and the `flex` parameter only supporting a single number.
 
+### Layout Direction
+
+Layout direction specifies the direction in which children and text in a hierarchy should be laid out. Layout direction also affects what edge `start` and `end` refer to. By default Yoga lays out with LTR layout direction. In this mode `start` refers to left and `end` refers to right.
+
+- `LTR` (**default value**) Text and children and laid out from left to right. Margin and padding applied the start of an element are applied on the left side.
+
+- `RTL` Text and children and laid out from right to left. Margin and padding applied the start of an element are applied on the right side.
+
+### Flex
+
+[`flex`](https://facebook.github.io/react-native/docs/layout-props#flex) will define how your items are going to **“fight”** over the available space along your primary axis. Most of the time you will want your app container to be flex:1 to take all of the screen height. Space will be divided according to each element flex property. In the following example the red, yellow and the green views are all children in the container view that got flex:1. The red view got flex:1 , the yellow view got flex:2 and the green view got flex:3 . 1+2+3=6 which means that red view will get 1/6 of the space, the yellow 2/6 of the space and the red 3/6 of the space. I think you got it . From [wix](https://medium.com/wix-engineering/the-full-react-native-layout-cheat-sheet-a4147802405c) engineers
+
+See some examples of [wix](https://medium.com/wix-engineering/the-full-react-native-layout-cheat-sheet-a4147802405c) engineers
+![Flex](https://cdn-images-1.medium.com/max/800/1*PhCFmO5tYX_sZSyCd4vO3w.png)
+
 #### Flex Direction
 
+[`flexDirection`](https://facebook.github.io/react-native/docs/layout-props#flexdirection) controls the direction in which children of a node are laid out. This is also referred to as the main axis. The main axis is the direction in which children are laid out. The cross axis the the axis perpendicular to the main axis, or the axis which wrapping lines are laid out in.
+
+- `row` (**default value**) Align children from left to right. If wrapping is enabled then the next line will start under the first item on the left of the container.
```

Comment:
**Important: the default value in React Native is column**

This may be worth calling out in a not somewhere because it is a common issue people run into

Reply:
Thanks for seeing that big mistake, I do not know what I was thinking about 😵